.buildkite/pipeline.yml

```
@@ -209,6 +209,10 @@ steps:
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200secs --t_end 3days"
         artifact_paths: "sphere_held_suarez_rhoe_equilmoist/*"
 
+      - label: ":computer: held suarez (ρe_tot) equilmoist monin_obukhov Float64"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --surface_scheme monin_obukhov --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200secs --t_end 3days --FLOAT_TYPE Float64"
+        artifact_paths: "sphere_held_suarez_rhoe_equilmoist_monin_obukhov_FT64/*"
```

```suggestion
        artifact_paths: "sphere_held_suarez_rhoe_equilmoist_mo_ft64/*"
```
(or anything, it just needs to be the same as jobid)